submissions/usides/js-dom/dist/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Intercatpol</title>
+  <link rel="stylesheet" href="./css/reset.css">
+  <link rel="stylesheet" href="./css/main.css">
+</head>
+
+<body>
+  <header>
+    <div class="container">
+      <div class="header-left-gr">
+        <img src="./img/logo.svg" alt="Intercatpol logo">
+        <h1>Intercatpol <br> wanted criminals </h1>
+      </div>
+      <a href="#" class="menu-btn" id="menu-btn"><img src="./img/lines.svg" alt="Menu button"></a>
```

Comment:
It's better to replace these links with `button` tags. Semantically it will be more correct.

Reply:
Done